waspc/data/Generator/templates/react-app/src/utils.js

```diff
@@ -0,0 +1,3 @@
+export const errorMessage = (e) => {
+  return `Error: ${e.message} ${e.data && e.data.message ? '- Details: ' + e.data.message : ''}`
```

Comment:
Looking good! Btw there is this "new" thing in Javascript: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining .
So you can do `e?.data.message` instead of `e.data && e.data.message`.

Reply:
hah yes, I also think I haven't used it yet :)

Reply:
Sweet! I loved that in Ruby and am glad JS has an equivalent! Thanks I will make use of that and update 😄

Reply:
This has been addressed here: https://github.com/wasp-lang/wasp/pull/376/commits/9a545b74ddf9be9e69eff94f13d72b19886f2acc Thanks!